x/staking/types/validator.go

```diff
@@ -400,14 +401,42 @@ func (v Validator) DelegatorShareExRate() sdk.Dec {
 	return sdk.NewDecFromInt(v.Tokens).Quo(v.DelegatorShares)
 }
 
-// Get the bonded tokens which the validator holds
+// get the bonded tokens which the validator holds
 func (v Validator) BondedTokens() sdk.Int {
 	if v.Status == sdk.Bonded {
 		return v.Tokens
 	}
 	return sdk.ZeroInt()
 }
 
+// get the Tendermint Power
+// a reduction of 10^9 from validator tokens is applied
+func (v Validator) TendermintPower() int64 {
+	if v.Status == sdk.Bonded {
+		return v.PotentialTendermintPower()
+	}
+	return 0
+}
+
+var powerReduction = sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(6), nil))
```

Comment:
Can `6` be a constant elsewhere defined - we'll probably want to use that in more places?

Reply:
I don't think we actually will need this elsewhere... if you need to convert between tokens and power you can just use the already exported `TokensToTendermintPower` or `TokensFromTendermintPower` functions

Reply:
@rigelrozanski You forgot to change comment about tendermint power from 9 to 6

Reply:
can you be more specific? I cannot find any comment referencing tendermint power and `9`

Reply:
https://github.com/cosmos/cosmos-sdk/blob/develop/x/staking/types/validator.go#L419

Reply:
@filippetroviccc ty! included this fix in this related PR https://github.com/cosmos/cosmos-sdk/pull/3654